pkg/apis/vpc/v1alpha1/addressbinding_types.go

```diff
@@ -7,6 +7,8 @@ type AddressBindingSpec struct {
 	VMName string `json:"vmName"`
 	// InterfaceName contains the interface name of the VM, if not set, the first interface of the VM will be used
 	InterfaceName string `json:"interfaceName,omitempty"`
+	// AllocatedExternalIPPath contains path of the allocated IP address.
+	AllocatedExternalIPPath string `json:"allocatedExternalIPPath,omitempty"`
```

Comment:
Do we need to expose `Path` in `IPAddressAllocationStatus`?

Reply:
I feel no need, since the Path is already under Spec.
If there is no AllocatedExternalIPPath from Spec, no need to set it under Status.

Reply:
for CR changes，could you also let Jianjun review?